### PR TITLE
Templates revision - part 2

### PR DESCRIPTION
**Pull request by Gwenwyfar**
Some more formatting and beginning work on html/css cleanup and fixes.

### CODE REVIEW

File: Themes/default/BoardIndex.template.php [hunk 1]
```diff
@@ -365,7 +365,7 @@ function template_ic_block_recent()
 		// latest_post has link, href, time, subject, short_subject (shortened with...), and topic. (its id.)
 		echo '
 				<p id="infocenter_onepost" class="inline">
-					<a href="', $scripturl, '?action=recent">', $txt['recent_view'], '</a>&nbsp;&quot;', sprintf($txt['is_recent_updated'], '&quot;' . $context['latest_post']['link'], '&quot;'), ' (', $context['latest_post']['time'], ')<br>
```
Comment (Gwenwyfar) [inline Themes/default/BoardIndex.template.php:370]: This one is also a minor fix for the end quote.

**Review by XinYenFon — APPROVED** [review]
Unless I missed some typo, LGTM.

**Comment by colinschoen** [comment]
I’ll review today. Sorry for delay.

File: Themes/default/Errors.template.php [hunk 2]
```diff
@@ -147,19 +145,19 @@ function template_error_log()
...
-							<span style="display: table-cell;">', $error['message']['html'], '</span>
+							<a class="error_message" href="', $scripturl, '?action=admin;area=logs;sa=errorlog', $context['sort_direction'] == 'down' ? ';desc' : '', ';filter=message;value=', $error['message']['href'], '" title="', $txt['apply_filter'], ': ', $txt['filter_only_message'], '"><span class="generic_icons filter"></span></a>
+							<span class="error_message">', $error['message']['html'], '</span>
 						</div>
 
-						<div style="float: left; width: 100%; padding: 4px 0; line-height: 1.6em; border-top: 1px solid #e3e3e3;">
-							<a style="display: table-cell; padding: 4px; width: 20px; vertical-align: top;" href="', $scripturl, '?action=admin;area=logs;sa=errorlog', $context['sort_direction'] == 'down' ? ';desc' : '', ';filter=url;value=', $error['url']['href'], '" title="', $txt['apply_filter'], ': ', $txt['filter_only_url'], '"><span class="generic_icons filter"></span></a>
-							<a style="display: table-cell;" href="', $error['url']['html'], '">', $error['url']['html'], '</a>
+						<div class="error_location">
```
Comment (colinschoen) [inline Themes/default/Errors.template.php:152]: screen shot 2017-12-06 at 15 25 14

**Current Error Log Table ^**

screen shot 2017-12-06 at 15 25 19

**Changed Error Log Table ^**

The additional cells help distinguish the different pieces of information. I find it harder to sort through the error log with everything aligned in one cell. Was this a deliberate style change?

**Review by colinschoen — CHANGES_REQUESTED** [review]
It seems like a lot more inline styles were removed than added to `.css` files. Were these CSS properties just not necessary?

**Comment by Gwenwyfar** [comment]
Yes, the inline styles that were not replaced were not doing anything, a few were even breaking things.

File: Themes/default/Errors.template.php (hunk 2 above)
Comment (Gwenwyfar) [reply Themes/default/Errors.template.php:152]: Hmm, did you include admin.css changes? That's not how it should look.

Comment (colinschoen) [reply Themes/default/Errors.template.php:152]: Let me double check. Perhaps the .css was cached and I didn't force refresh after checking out the PR branch.

Comment (colinschoen) [reply Themes/default/Errors.template.php:152]: And you are correct, thanks.

**Review by colinschoen — APPROVED** [review]
Pending merge conflicts fixed:

LGTM approval

**Pull request merged**